pkg/serverless/logs/logs_test.go

```diff
@@ -673,65 +673,70 @@ func TestProcessLogMessageLogsNotEnabled(t *testing.T) {
 }
 
 func TestProcessLogMessagesTimeoutLogFromReportLog(t *testing.T) {
-	logChannel := make(chan *config.ChannelMessage)
```

Comment:
Most of this huge diff is just adding a subtest with `t.Run`, which required indenting the whole test.